src/Listeners/RefreshInventory.php

```diff
@@ -27,7 +27,7 @@ public function handle(OrderPlaced $event): void
             }
 
             return $shouldReject;
-        })->groupBy(function (Model $product): string {
```

Comment:
This can be a product or a variation. Don't change this, please. The other changes are OK, thanks.

Reply:
Done.